### PR TITLE
filter out arm64, macOS, and crio from flake reports

### PR DESCRIPTION
**Pull request by sharifelgamal**
We want to filter these out of PR comments until they're more stable and less noisy

### CODE REVIEW

File: hack/jenkins/test-flake-chart/report_flakes.sh [hunk 1]
```diff
@@ -85,6 +85,10 @@ awk -F, 'NR>1 {
   | sort -g -t, -k2,2 \
   >> "$TMP_FAILED_RATES"
 
+# Filter out arm64 and macOS tests until they're more stable
+grep -v arm64 "$TMP_FAILED_RATES" > "$TMP_FAILED_RATES"
+grep -v macOS ""$TMP_FAILED_RATES" > ""$TMP_FAILED_RATES"
```
Comment (spowelljr) [inline hack/jenkins/test-flake-chart/report_flakes.sh:90]: I'm assuming the extra double quotes are a mistake?

Comment (sharifelgamal) [reply hack/jenkins/test-flake-chart/report_flakes.sh]: yeah i fixed it

File: hack/jenkins/test-flake-chart/report_flakes.sh [hunk 2]
```diff
@@ -85,6 +85,10 @@ awk -F, 'NR>1 {
   | sort -g -t, -k2,2 \
   >> "$TMP_FAILED_RATES"
 
+# Filter out arm64 and macOS tests until they're more stable
+grep -v arm64 "$TMP_FAILED_RATES" > "$TMP_FAILED_RATES"
+grep -v macOS "$TMP_FAILED_RATES" > "$TMP_FAILED_RATES"
```
Comment (spowelljr) [inline hack/jenkins/test-flake-chart/report_flakes.sh:90]: Could do grep -v "arm64\|macOS" to reduce some duplication but not required

**Review by spowelljr — APPROVED** [review]

File: hack/jenkins/test-flake-chart/report_flakes.sh (hunk 2 above)
Comment (sharifelgamal) [reply hack/jenkins/test-flake-chart/report_flakes.sh:90]: yeah, that's definitely true, I just wanted to make it super clear and allow for easy deletion.

File: hack/jenkins/test-flake-chart/report_flakes.sh [hunk 3]
```diff
@@ -85,7 +85,11 @@ awk -F, 'NR>1 {
   | sort -g -t, -k2,2 \
   >> "$TMP_FAILED_RATES"
 
-FAILED_RATES_LINES=$(wc -l < "$TMP_FAILED_RATES")
+# Filter out arm64, macOS and windows tests until they're more stable
+TMP_FAILED_RATES_FILTERED=$(mktemp)
+grep -v "arm64\|macOS\|Windows" "$TMP_FAILED_RATES" > "$TMP_FAILED_RATES_FILTERED"
```
Comment (medyagh) [inline hack/jenkins/test-flake-chart/report_flakes.sh:90]: I think instead of filtering out Windows

lets filterout Crio and Arm64

**Review by medyagh — CHANGES_REQUESTED** [review]

File: hack/jenkins/test-flake-chart/report_flakes.sh (hunk 3 above)
Comment (sharifelgamal) [reply hack/jenkins/test-flake-chart/report_flakes.sh]: done

**Pull request merged**